modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -328,6 +328,26 @@ AUI.add(
 				return value || '';
 			},
 
+			_getType: function(value, options) {
+				var instance = this;
+
+				var option;
+
```

Comment:
Remove this line break space between the variables declaration: `option` and `type`

Reply:
ok